rackcity/api/serializers/fields/rackcity_serializer_fields.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework.serializers import IntegerField
+from rest_framework.fields import (
+    lazy_format,
+    MaxValueValidator,
+    MinValueValidator
+)
+
+
+class RCIntegerField(IntegerField):
+
+    def __init__(self, **kwargs):
+        self.max_value = kwargs.pop('max_value', None)
+        self.min_value = kwargs.pop('min_value', None)
+        super().__init__(**kwargs)
+        if self.max_value is not None and self.max_value != "":
```

Comment:
Had to override this method because the "" broke comparison to min and max values. Only change is from `if self.max_value is not None:` to `if self.max_value is not None and self.max_value != "":`